tools/qdstat.in

```diff
@@ -155,7 +155,19 @@ class BusManager(Node):
             return text[:-1]
         return text
 
-    def displayEdges(self):
+    def displayRouterId(self):
+        objects = self.query('org.apache.qpid.dispatch.router')
+        router = objects[0]
+        if router:
+            print (router.id)
```

Comment:
Please remove the space between the print and it's argument list.
This is not PEP8 compliant and will cause python lint checkers to flag it as a violation.